wiki/Do_you_really_want_to_ask_peppy/ro.md

```diff
@@ -0,0 +1,15 @@
+# Chiar dorești să-l întrebi pe peppy?
+
+Salutare! peppy înțelege dorința ta de a dori să-l contactezi personal, dar te roagă să înțelegi mai întâi că acesta **nu** este locul unde vrei să-i ceri ajutor sau sprijin. El alege să păstrează asistența osu! separat de mesajele personale și de asemenea, consideră că este destul de greu să răspundă la PM-urile de pe forum. Prin urmare, vă solicită să faceți una dintre următoarele opțiuni:
+
+## Suportul tehnic
+
+Majoritatea problemelor tehnice pot fi rezolvate căutând sau postând în secțiunea [Forum-ul de ajutor](https://osu.ppy.sh/community/forums/5). Sunt mulți alții acolo care te pot ajuta cu problemele tale, așa că vei găsi o soluție mai repede decât să-l contactezi pe el.
```

Comment:
"Majoritatea problemelor tehnice pot fi rezolvate căutând un răspuns sau postând o întrebare în secțiunea [Help forum](https://osu.ppy.sh/community/forums/5). Sunt mulți alți utilizatori acolo care te pot ajuta cu problemele tale, cel mai probabil vei găsi o soluție mai repede acolo, decât contactându-l pe peppy."

Help forum nu este tradus pe nici-o versiune a site-ului deci cel mai bine să se facă referire la denumirea în limba engleză.

Reply:
De fapt e tradus aici https://osu.ppy.sh/wiki/ro/Main_Page
ca de aici l-am si luat ca sa fie o consistenta

Reply:
nu este adevarat. verifica: https://osu.ppy.sh/wiki/ro/Main_Page

help forum este tradus, de asemenea, pe crowdin in schimbarile care urmeaza luna asta

Reply:
Atunci ramane doar sugestia mea pt rephrasing generala: 

`Majoritatea problemelor tehnice pot fi rezolvate căutând un răspuns sau postând o întrebare în secțiunea [Forum-ul de ajutor](https://osu.ppy.sh/community/forums/5). Sunt mulți alți utilizatori acolo care te pot ajuta cu problemele tale, cel mai probabil vei găsi o soluție mai repede acolo, decât contactându-l pe peppy.`

Reply:
poti adauga o sugestie separata pt modificarea asta atunci, te rog?

Reply:
ca si paralela de asta ziceam ca e bine sa zici ce modificari ai facut la fiecare sugestie, ca nu observ la prima vedere daca tot ce ai sugerat e legat de help forum sau nu lol